packages/storybook/stories/Form.stories.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react'
+import customStyles from '../stc.config'
+
+const meta: Meta<typeof Form> = {
+  title: 'Components/Form',
+  component: Form,
+  parameters: {
+    backgrounds: {
+      default: 'Dark',
+    },
+  },
+  argTypes: {
+    id: {
+      type: 'string',
+      value: 'formId',
+    },
+    debug: {
+      type: 'boolean',
+      value: true,
+    },
+    defaultValues: {
+      key: { type: 'string', value: 'any' },
+    },
+    mode: {
+      type: 'string',
+      control: 'select',
+      options: ['onBlur', 'onError', 'onChange', 'onSubmit', 'all'],
+    },
+    title: {
+      type: 'string',
+      value: 'fieldset',
+    },
+    withPanel: {
+      type: 'boolean',
+      value: false,
+    },
+    customStyles: {
+      // type: "object",
+      value: {},
+    },
+  },
+}
+
+export default meta
+
+const FieldsetTemplate: StoryFn<DateInputProps> = (args) => (
+  <Form {...args} onSubmit={(data: unknown) => console.log(data)}>
+    <Select
+      name="dropdown"
+      label="Dropdown"
+      required
+      options={[
+        { value: 'male', label: 'Male' },
+        { value: 'female', label: 'Female' },
+      ]}
+    />
+    <Fieldset label="Personal Details">
+      <Text
+        name="personal_details.firstName"
+        placeholder="John"
+        label="First Name"
+        required
+      />
+      <Text
+        name="personal_details.lastName"
+        placeholder="Doe"
+        label="Last Name"
+        required
+      />
+      <RadioButtonGroup
+        name="personal_details.gender"
+        label="Gender"
+        required
+        radioButtons={[
+          { value: 'male', label: 'Male' },
+          { value: 'female', label: 'Female' },
+        ]}
+      />
+    </Fieldset>
+    <Fieldset label="Contact Details">
+      <Email
+        name="contact_details.email"
+        placeholder="steve@sentosatech.com"
+        label="Email"
+        required
+      />
+      <Text
+        name="contact_details.phone"
+        placeholder="+1 99999999"
+        label="Phone Number"
+        required
+      />
+    </Fieldset>
+    <DateInput
+      name="birthDate"
+      placeholder="07/22/1993"
+      label="Birth Date"
+      required
+    />
+    <NumberInput name="age" placeholder="Age" label="Age" required />
+    <TimeInput name="time" placeholder="Time" label="Time" required />
+    <TextArea
+      name="address"
+      label="Street Address"
+      placeholder="Street Address"
+      required
+    />
+    <RadioButtonGroup
+      name="option"
+      label="Option"
+      required
+      direction="vertical"
+      radioButtons={[
+        { value: 'option 1', label: 'Option 1' },
+        { value: 'option 2', label: 'Option 2' },
+        { value: 'option 3', label: 'Option 3' },
+        { value: 'option 4', label: 'Option 4' },
+      ]}
+    />
+    <CheckBox name="agree" label="Agree to Terms and Conditions" required />
```

Comment:
<img width="367" alt="Screenshot 2024-11-01 at 9 27 46 AM" src="https://github.com/user-attachments/assets/78d5c8b2-3d9e-4d9e-990d-9f94cb13341f">

@ianmarquez minor thing here, but can we troubleshoot the style so that the error is rendered below the input?